assets/js/main.js

```diff
@@ -2,11 +2,13 @@ import $ from 'jquery';
 import ShowHideContent from 'govuk/show-hide-content';
 import accessibleAutocomplete from 'accessible-autocomplete';
 import Analytics from 'govuk/analytics/analytics';
+import datePicker from './date-picker';
+
```

Comment:
It's not enough just to import the date picker, you should at least perform some initialisation